### PR TITLE
CI Docker

### LINKED ISSUE #563: Review of pipelines executed after push
Review task of test pipelines executed after pushing changes into the repository. They generate errors in most of the correct changes pushed

### PR DESCRIPTION
**Pull request by vcerenu**
This PR modifies the Github Actions script in which the branches in which a Pull Request is created are tested, to verify that there are no errors in the creation of the Docker images and the subsequent deployment of the stacks.
Closes #563

### CODE REVIEW

**Review by teddytpc1 — DISMISSED** [review]
LGTM

**Review by teddytpc1 — DISMISSED** [review]
LGTM

**Review by teddytpc1 — APPROVED** [review]
LGTM

File: .github/.goss.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ package:
   wazuh-manager:
     installed: true
     versions:
-    - 4.3.4
+    - 4.3.4-1
```
Comment (okynos) [inline .github/.goss.yaml:59]: It is 4.3.3 or 4.3.4?

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+WAZUH_VERSION=4.3.3
```
Comment (okynos) [inline .env:1]: Same

**Review by okynos — CHANGES_REQUESTED** [review]

**Review by alberpilot — APPROVED** [review]

**Pull request merged**